universal-application-tool-0.0.1/app/models/QuestionTag.java

```diff
@@ -0,0 +1,20 @@
+package models;
+
+import io.ebean.annotation.DbEnumType;
+import io.ebean.annotation.DbEnumValue;
+
+public enum QuestionTag {
```

Comment:
suggestion (blocking): please document this public class

Reply:
Done.